autogpt_platform/backend/backend/blocks/vector_dbs/pinecone.py

```diff
@@ -0,0 +1,117 @@
+from typing import Literal
+
+from autogpt_libs.supabase_integration_credentials_store import Credentials
+from pinecone import Pinecone, ServerlessSpec
+
+from backend.data.block import Block, BlockCategory, BlockOutput, BlockSchema
+from backend.data.model import BlockSecret, SchemaField, SecretField, CredentialsMetaInput, CredentialsField
+
+
+class PineconeInitBlock(Block):
+    class Input(BlockSchema):
+        credentials: CredentialsMetaInput[Literal['pinecone'], Literal['api_key']] = CredentialsField(
+            provider="pinecone",
+            supported_credential_types={"api_key"}, # noqa
```

Comment:
```suggestion
            supported_credential_types={"api_key"},
```

Reply:
does it not give your warnings without it?

Reply:
not at all